build/syntax.go

```diff
@@ -361,6 +361,22 @@ func (x *BinaryExpr) Span() (start, end Position) {
 	return start, end
 }
 
+// An AssignmentExpr represents a binary expression with `=`: X = Y.
+type AssignmentExpr struct {
+	Comments
+	X         Expr
+	OpStart   Position
+	Op        string
+	LineBreak bool // insert line break between Op and Y
+	Y         Expr
+}
```

Comment:
Use the same definition as https://github.com/google/starlark-go/blob/master/syntax/syntax.go?

Reply:
I'll rename the fields, but the nodes won't be equivalent: in stalark-in-go keyword parameters are still BinaryExpr, in this PR they become AssignmentExpr. I did that intentionally, the linter often needs to distinguish between random binary expressions (e.g. `a + b`) and keyword parameters(`f(a = b)`). This also technically affects some weird cases that don't exist in real code (and can't be generated by the parser) like `f(a += b)`.

Reply:
Changed the definition.